test/routes/default.js

```diff
@@ -27,7 +27,8 @@ module.exports = [{
             query: Joi.object({
                 param1: Joi.string().required()
             })
-        }
+        },
+        tags: ['testTag']
```

Comment:
To make significant tests I'd recommend adding the same tag to several routes just to make sure we get all of them.